course/pages/dagster-etl/lesson-5/5-refactoring-apis-with-dlt.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Lesson 5: Refactoring apis with dlt"
+module: 'dagster_etl'
+lesson: '5'
+---
+
+# Refactoring APIs with dlt
+
+TODO - Determine dlt code is correct
```

Comment:
Happy to meet to discuss this. We can also call out how we use it in dagster-open-platform.

https://github.com/dagster-io/dagster-open-platform/blob/6246cfa3f8f9171debd1dc6158909b14adce879a/dagster_open_platform/defs/dlt/sources/buildkite/__init__.py#L58-L95